modules/database/src/std/rec/compressRecord.c

```diff
@@ -149,13 +149,16 @@ static int compare(const void *arg1, const void *arg2)
     else               return  1;
 }
 
+#define min(a, b) ((a) < (b) ? (a) : (b))
```

Comment:
Duplicate.

Reply:
I actually can't find another definition anywhere for this. The only spot I can find similar definitions are:

https://github.com/epics-base/epics-base/blob/7.0/modules/libcom/src/flex/flexdef.h#L70

https://github.com/epics-base/epics-base/blob/7.0/modules/database/src/ioc/db/dbTest.c#L57

neither of which seems a good fit to try to include here.

Reply:
In epicsAlgorithm.h we defined `epicsMin()` (which is now deprecated in favor of `std::min()`), and Jeff Hill's even older tsMinMax.h defined `tsMin()`, but they are of course both for use from C++ only — sorry, my mistake there!